lib/mqtt/mqtt_client.dart

```diff
@@ -0,0 +1,141 @@
+import "package:flutter/foundation.dart";
+import "package:flutter_dotenv/flutter_dotenv.dart";
+import "package:mqtt_client/mqtt_client.dart";
+import "package:mqtt_client/mqtt_server_client.dart";
+import "package:oss_surveys_customer/mqtt/model/status_message.dart";
+import "package:typed_data/typed_buffers.dart";
+import "../main.dart";
+
+/// MQTT Client
+class MqttClient {
+  
+  late final MqttServerClient _client;
+  final statusTopic = "oss/status";
+  
+  /// Public constructor.
+  /// 
+  /// Initializes [MqttServerClient]
+  MqttClient() {
+    var mqttBasePath = dotenv.env["MQTT_URL"];
+    var mqttClientId = dotenv.env["MQTT_PORT"];
```

Comment:
Did you mean to use mqtt_client_id here? Seems like a typo

Reply:
True. Resolved